### DISCUSSION TITLE
微信支付服务商下的“特约商户”审核API有吗？

### DISCUSSION THREAD

**Discussion by GitHub-Ahai** [w7corp/easywechat#2616] [Q&A]
微信支付服务商下的“特约商户”审核API有吗？日常都是手动去人工到后台上传审核太麻烦了

**Comment by GitHub-Ahai** [comment]
看了很久文档没有找到，请问 服务商下的“特约商户” 是小微商户吗？

**Reply by TheNorthMemory** [reply]
**特约商户**的概念，大于**小微商户**，**小微**一般指连「个体工商户」都不是的个人商户，详细建议阅读 https://kf.qq.com/faq/200114EZfmy62001142YVRzy.html 及 https://kf.qq.com/faq/220520uIZfqQ220520RJFJNv.html

**Reply by GitHub-Ahai** [reply]
“特约商户”审核API 专业术语 【特约商户进件】这个扩展包有提供吗？

**Reply by TheNorthMemory** [reply]
v6版，你可以自己构造URI就有，若你都不清楚URI是什么，那就乏天无数了。

PS: 审核API一般是机构服务商才有的吧？

**Reply by GitHub-Ahai** [reply]
是的 机构服务商

**Comment by Memory-9527** [comment]
<img width="1349" alt="image" src="https://user-images.githubusercontent.com/30432849/199391822-01e6c384-2290-45b0-9ceb-3cf561ebbd6f.png">
文档中心第一个不就是进件接口么，你怎么看的文档。。。。。。

**Reply by GitHub-Ahai** [reply]
我问的是这个扩展包有没有更加快速的...
